src/test/config.sol

```diff
@@ -95,9 +95,9 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x04D0E226675cfC2450D523C961c1F1875f20bf47),         // populate with deployed spell if deployed
-            deployed_spell_created:         1667323871,                  // use `./scripts/get-created-timestamp.sh <deployment-tx>`
-            previous_spell:                 address(0),         // supply if there is a need to test prior to its cast() function being called on-chain.
+            deployed_spell:                 address(0),         // populate with deployed spell if deployed
+            deployed_spell_created:         0,                  // use `./scripts/get-created-timestamp.sh <deployment-tx>`
+            previous_spell:                 address(0x04D0E226675cfC2450D523C961c1F1875f20bf47),         // supply if there is a need to test prior to its cast() function being called on-chain.
```

Comment:
This spell is already executed so we could set this up to `address(0)`.

Reply:
Done.